README.md

```diff
@@ -4,8 +4,16 @@ Persistent cache for facility lookups.
 
 ## Deployment
 
+### Command line
+
 1. Clone this repository.
 1. Run `npm install`.
 1. Create a config file e.g. `production.json` in the `config` directory. See the [default config](https://github.com/jembi/facility-cache/blob/master/config/config.json) for the available options.
 1. Make sure that your `NODE_ENV` environment variable is set correctly e.g. `export NODE_ENV=production`.
 1. Run `npm start`.
+
```

Comment:
@Zooloo2014 Could you possibly add the docker build instruction here somewhere as well - just to be thorough